app/serializers/api/public/v1/serializable_course.rb

```diff
@@ -18,14 +18,14 @@ def enrichment_attribute(name, enrichment_name = name)
 
         type 'courses'
 
-        belongs_to :accredited_body do
+        belongs_to :accredited_provider do
```

Comment:
i don't think we should change any of the serializers otherwise we would be introducing breaking changes for both apply and register for their courses data sync scripts